xServerWorker/Jobs/PayXServersJob.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Common.Models.x42Blockcore;
+using Common.Models.XServer;
+using Quartz;
+using RestSharp;
+using System.Diagnostics;
+
+namespace xServerWorker.Jobs
+{
+    [DisallowConcurrentExecution]
+    public class PayXServersJob : IJob
+    {
+        public PayXServersJob()
+        {
+        }
+        public async Task Execute(IJobExecutionContext context)
+        {
+            using (var loggerFactory = LoggerFactory.Create
+          (
+              builder =>
+              {
+                  builder.AddConsole();
+              }
+          ))
+            {
+                Console.Write("Pay XServer");
+
+                var _logger = loggerFactory.CreateLogger<PayXServersJob>();
+
+#if DEBUG
+                var x42BlockCoreClient = new RestClient("http://localhost:42220/api/");
+#else
+ var x42BlockCoreClient = new RestClient("http://x42core:42220/api/");
+#endif
+
+                 
+                var xServerClient = new RestClient("http://144.91.95.234:4242/");
```

Comment:
xServerClient hardcoded IP address

Reply:
Still not completed, will update once done